modules/axhal/src/platform/aarch64_qemu_virt/mod.rs

```diff
@@ -20,8 +20,6 @@ pub mod misc {
     pub use crate::platform::aarch64_common::psci::system_off as terminate;
 }
 
-//Todo: remove this, when hv is enabled, `exception_vector_base` is not used.
```

Comment:
What does this todo mean?

Reply:
`exception_vector_base` is not used when "hv" feature is enabled in the previous design. Exception vector base was registered by vmm-app through methods like `register_lower_aarch64_irq_handler`. 

`#[cfg_attr(feature = "hv", allow(unused))]` is used here to bypass clippy warning about unused variables. It can be removed happily now.